### PR TITLE
Mobile Portfolio Version

### PR DESCRIPTION
**Pull request by sainingo**
# Mobile Portfolio Version 
> sainingo portfolio mobile version

The portfolio is designed using html and css flex
- It based on the figma template 3 design
- It basically contain 2 main section 
  - the header which contains the welcome tag and hamburger menu
  - And the headline section which is just a section
*Please let me know If there's anything i missed out*

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>Portfolio</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <script src="https://kit.fontawesome.com/f41acb8d77.js" crossorigin="anonymous"></script>
+        <link rel="stylesheet" href="styles.css">
+    </head>
+    <body>
+          <!-- Header section -->
```
Comment (elmar8287) [inline index.html:13]: - [ ] [OPTIONAL] Please, consider removing all comments to make your code cleaner 👍

**Review by elmar8287 — APPROVED** [review]
Hi @sainingo, 

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 

# Please, before merging, fix the OPTIONAL comment under review! 👍 💯 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (sainingo) [reply index.html:13]: Ok will do that before merging  and thanks for reviewing my code @elmar8287

Comment (elmar8287) [reply index.html:13]: > Ok will do that before merging and thanks for reviewing my code @elmar8287

👍

**Pull request merged**